### DISCUSSION TITLE
Introdction

### DISCUSSION THREAD

**Discussion by prajjaldhar** [community/community#47157] [GitHub Education]
Hello coders!!!! This side Prajjal Dhar.....Recently being part of this community... I hope that, by participating in such programs, I will be able to have a positive impact on my community and create lasting relationships with the people around me

**Comment by Ntshangase** [comment]
Hi Prajjal,

If you want to start earning badges to add to your profile checkout [https://github.com/Schweinepriester/github-profile-achievements](url) .And if you found this helpful don't forget to mark this as answer because I'm also trying to earn more badges.
